bots/tests-scan

```diff
@@ -199,6 +199,7 @@ def scan_for_pull_tasks(api, update, human, policy):
             return False
         return True
 
+    baseline = BASELINE_PRIORITY - 3
```

Comment:
since `baseline` is also used for contexts later on, could we rename this variable to `branch_baseline` or `baseline_branch`?